packages/realm/src/binding.ts

```diff
@@ -50,6 +66,23 @@ declare module "realm/binding" {
   }
 }
 
+Geospatial.fromCircle = function (c: ExternalGeoCircle): Geospatial {
+  return Geospatial.makeFromCircle({
+    center: convertToBindingGeoPoint(c.center),
+    radiusRadians: c.distance,
+  });
+};
+
+function convertToBindingGeoPoint(p: ExternalGeoPoint): BindingGeoPoint {
```

Comment:
Small thing, but our functions for converting to/from a binding type are usually just called `toBinding<Type>` (e.g. `toBindingGeoPoint`) 🙂